# Li2022
Full title: *Exaggerated false positives by popular differential expression methods when analyzing human population samples*
Authors: *Yumei Li* (first); *Jingyi Jessica Li* (last)
Year: *2022*
Topic: #ComputatinalBiology #RNAseq #DEA
Journal: *Genome Biology*
DOI: https://doi.org/10.1186/s13059-022-02648-4
Date: 2022-10-18

---

Source: The original source of the document can be found [here](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02648-4#citeas)

---

## Summary

When the per-condition sample size is *less than 8*, parametric methods for *DE analysis*  like DEseq2 or edgeR can be used bacaue their *power advantage outwheights their possibly exaggerated false positives*. if concerned about *FDR control*, seems that the *Wilcoxon rank-sum test* is the only one that can effectively control it

## Index
[[00_Index_papers#^6b8c22]]