
# RNAseq
Topic: #RNAseq #Omics #Transcriptomics 
Date: 2022-08-30

---

## Summary
Changed the transcriptomics field. A cost-effective way to understand gene function is the quantification of the abundance of a gene/transcript. Usually, tho it doesn't reflect the whole complexity of a functional molecule, number of copies of a gene's product is considered a quantitative measure of gene's activity. RNAseq is used to profile at a single nucleotide resolution, the abundace of tens of thousands of distinct trascripts encoded in the genome. RNAseq provides data that's valuable and contains info on exons and transcripts

## Notes
- With RNAseq you can quanitfy a*t transcript-level* or *gene-level*, transcript being less precise due to much less unique sequence, mainly because different trancripts from the same gene can containt same exons or UTR regions.
- Common *biases* : fragmentation, amplification sequencing process.
- *Alignment-dependent* methods and *aligner-independet* methods.
- *Pseudoaligners* like [[Kallisto]] can rapidly determine the set of transcripts that are compatible with a given seqeunced fragment.
-  [[Salmon_RNAseq]], is similar to a pseudoaligner but use a differen fast and lightweight model called *quasi-mapping* [[Quasi-mapping]]

## Questions
- Item



