
# VanGalen_SEMMinars
Topic: #Leukimia #SingleCell 
Date: 2022-09-29

---

## Summary
Notes on leukimia tracking w/ single cell methods 

## Notes
- Multimodal single cell genomics
- *3' barcoded cDNA* singe cell protocol to do transcriptome, nucler genome mutations, mitochondrial dna mutations and T celll receptor.
- *SPLiT-seq* and *sci-RNAseq* `check them out`.
- Pretty hard to distinguish malignat from healthy cells, they used Biotin-PCR target enritchment to capture mutations, ~10% efficiency. Pretty inefficient as he says.
- [Bradley Bernstein](https://bernstein.dfci.harvard.edu/)
- *Heteroplasmic threshold*
- Use mutations to infer the donor origin when mixing different cancer cells.
- [MAESTER protocol](https://www.rna-seqblog.com/maester-mitochondrial-variant-enrichment-from-high-throughput-single-cell-rna-sequencing/) 
- Challenging to assess different clones in the same donor
- pipeline for TCR enrichment form 3' single cell seq, [WAT3R](https://github.com/mainciburu/WAT3R ),
- [scPred](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1862-5) and [cellHarmony](https://github.com/nsalomonis/altanalyze/wiki/cellHarmony) as annotating algorithm for cell type in the patients.


## Questions
- Item



